lib/datadog/tracing/configuration/settings.rb

```diff
@@ -437,6 +437,15 @@ def self.extended(base)
                 o.default { env_to_int(Tracing::Configuration::Ext::Distributed::ENV_X_DATADOG_TAGS_MAX_LENGTH, 512) }
                 o.lazy
               end
+
+              # Schema version for span attributes that enables various features
+              #
+              # @default `DD_TRACE_SPAN_ATTRIBUTE_SCHEMA` environment variable, otherwise `v0`
+              # @return [String,nil]
```

Comment:
I don't think we need to allow empty values here:
```suggestion
              # @return [String]
```